DataConnectors/Corelight/Connector_LogAnalytics_agent_Corelight.json

```diff
@@ -0,0 +1,148 @@
+{
+    "id": "Corelight",
+    "title": "Corelight",
+    "publisher": "Corelight",
+    "descriptionMarkdown": "The [Corelight](https://corelight.com/) data connector provides the capability to ingest [Corelight Zeek/Bro events](https://www3.corelight.com/zeek-logs-v3.0) into Azure Sentinel. Refer to [Corelight Logs documentation](https://corelight.com/about-zeek/how-zeek-works) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **Corelight** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-Corelight-parser)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Corelight Events",
+            "baseQuery": "Corelight"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Clients (Source IP)",
+            "query": "Corelight\n | summarize count() by SrcIpAddr\n | top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "Corelight_CL",
+            "lastDataReceivedQuery": "Corelight\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "Corelight\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1

```

Comment:
Status should be 2. Please refer sample template https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_REST_API_template.json

Thanks,
Aditi Maheshwari

Reply:
done